<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_graph_history</class>
 <widget class="QDialog" name="Dialog_graph_history">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>770</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>981</width>
    <height>770</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="Distance">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Distancia</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCustomPlot" name="graph_h" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>751</width>
             <height>556</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>181</width>
             <height>561</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>181</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QFrame" name="frame_3">
            <property name="geometry">
             <rect>
              <x>-1</x>
              <y>140</y>
              <width>181</width>
              <height>415</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>181</width>
              <height>400</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="label_6">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>170</y>
               <width>161</width>
               <height>20</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Medidas:</string>
             </property>
            </widget>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>30</y>
               <width>101</width>
               <height>20</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ID 1:</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>151</width>
               <height>20</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Medidas:</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBox_1">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>28</y>
               <width>21</width>
               <height>26</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBox_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>128</y>
               <width>21</width>
               <height>26</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>130</y>
               <width>101</width>
               <height>20</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ID 2:</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_7">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>270</y>
               <width>161</width>
               <height>20</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Medidas:</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>230</y>
               <width>101</width>
               <height>20</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ID 3:</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBox_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>228</y>
               <width>21</width>
               <height>26</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QLabel" name="label_8">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>370</y>
               <width>151</width>
               <height>20</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Medidas:</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBox_4">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>328</y>
               <width>21</width>
               <height>26</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>330</y>
               <width>111</width>
               <height>20</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ID 4:</string>
             </property>
            </widget>
            <widget class="Line" name="line">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>181</width>
               <height>16</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
            <widget class="Line" name="line_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>100</y>
               <width>181</width>
               <height>16</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
            <widget class="Line" name="line_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>200</y>
               <width>181</width>
               <height>16</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
            <widget class="Line" name="line_4">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>300</y>
               <width>181</width>
               <height>16</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
            <widget class="Line" name="line_5">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>400</y>
               <width>181</width>
               <height>16</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
            <widget class="Line" name="line_6">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>3</y>
               <width>181</width>
               <height>16</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </widget>
           <widget class="QFrame" name="frame_4">
            <property name="geometry">
             <rect>
              <x>-1</x>
              <y>0</y>
              <width>181</width>
              <height>141</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>181</width>
              <height>141</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>181</width>
              <height>141</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="pushButton_2">
             <property name="geometry">
              <rect>
               <x>15</x>
               <y>70</y>
               <width>151</width>
               <height>29</height>
              </rect>
             </property>
             <property name="text">
              <string>Reiniciar mediciones</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton">
             <property name="geometry">
              <rect>
               <x>35</x>
               <y>30</y>
               <width>111</width>
               <height>29</height>
              </rect>
             </property>
             <property name="text">
              <string>Reiniciar zoom</string>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>949</width>
           <height>90</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>90</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>941</width>
            <height>71</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Rango seleccionado:</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_7">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Medidas:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Media:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Máximo:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Mínimo:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Positioning">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Posicionamiento</string>
      </attribute>
      <widget class="QCustomPlot" name="pos_h" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>941</width>
         <height>661</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>10</y>
          <width>111</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Reiniciar zoom</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>10</y>
          <width>111</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Reiniciar datos</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>50</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Medidas:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>50</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_graph_history</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_graph_history</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
